.github/workflows/yamllint.yml

```diff
@@ -0,0 +1,17 @@
+name: yamllint
+on: [pull_request]
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  yamllint:
+    name: check / yamllint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: yamllint
+        uses: reviewdog/action-yamllint@v1
+        with:
+          github_token: ${{ secrets.github_token }}
+          reporter: github-pr-review
```

Comment:
Please add the `fail_on_error: true` option too.
https://github.com/reviewdog/action-yamllint#fail_on_error